src/install/pre_install.sh

```diff
@@ -62,6 +62,9 @@ echo \
 sudo apt-get update
 sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
 
+sudo curl -SL https://github.com/docker/compose/releases/download/v2.20.0/docker-compose-linux-x86_64 -o /usr/local/bin/docker-compose
```

Comment:
This is not what the [installation instructions](https://github.com/docker/compose#linux) tell us.
But since we already install the docker repository we should just install it [from there](https://docs.docker.com/engine/install/ubuntu/).

Since docker compose v2 it should be invoked by `docker compose` instead of a separate binary `docker-compose`.
So remember to change that.

Please also keep/update the comment. I think its useful to know why we need our dependencies.